benchmarks/README.md

```diff
@@ -0,0 +1,15 @@
+# Benchmarks, to be run by codspeed.io in CI
```

Comment:
Not sure if this should be Markdown or RST.

I think that is important to mention that the tests are executed using `pytest` and not using trial.

This is why the test files are outside of `src/twisted`